qiskit/user_config.py

```diff
@@ -29,14 +30,17 @@ class UserConfig:
     [default]
     circuit_drawer = mpl
     circuit_mpl_style = default
+    circuit_mpl_style_path = /home/user/styles;/home/user
```

Comment:
I would go like something like this here, right?

```suggestion
    circuit_mpl_style_path = ~/.qiskit:<default location>
```

Reply:
Done in https://github.com/Qiskit/qiskit-terra/pull/5223/commits/bfb8be30672ba6d81187f57c58a4079026e0bb0e